modules/gapi/include/opencv2/gapi/render/render_types.hpp

```diff
@@ -133,23 +133,23 @@ struct Rect
      * @param lt_     The type of the line. See #LineTypes
      * @param shift_  The number of fractional bits in the point coordinates
      */
-    Rect(const cv::Rect& rect_,
-         const cv::Scalar& color_,
-         int thick_ = 1,
-         int lt_ = 8,
-         int shift_ = 0) :
+    GAPI_WRAP Rect(const cv::Rect2i& rect_,
```

Comment:
@dmatveev Do u mind replacing `cv::Rect` -> `cv::Rect2i` due it conflicts with `cv::gapi::wip::draw::Rect` in python generated code.